docs/platforms/javascript/common/configuration/integrations/localvariables.mdx

```diff
@@ -0,0 +1,33 @@
+---
+title: LocalVariables
+description: "Add local variables to exception frames. (default)"
+supported:
+  - javascript.node
+  - javascript.aws-lambda
+  - javascript.azure-functions
+  - javascript.connect
+  - javascript.express
+  - javascript.gcp-functions
+  - javascript.koa
+  - javascript.electron
+---
+
+_Import name: `Sentry.localVariablesSyncIntegration`_
+
+This integration is enabled by default. If you'd like to modify your default integrations, read [this](./../#modifying-default-integrations).
+
+The `onUnhandledRejectionIntegration` registers handlers to capture global unhandled promise rejections.
+
+## Options
+
+### `exitEvenIfOtherHandlersAreRegistered`
+
+_Type: `boolean`_
+
+If set to false, the SDK will _not_ exit if we detect that another uncaught exception handler is registered.
+
+### `onFatalError`
+
+_Type: `(firstError: Error, secondError: Error | undefined) => void`_
+
+This method is called when an uncaught error would cause the process to exit. `secondError` will be set if the handler was called multiple times. This can happen either because `onFatalError` itself threw, or because an independent error happened somewhere else while `onFatalError` was running.
```

Comment:
These lines seem to have been copied over from OnUnhandledRejection. If this was on purpose, please confirm